h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1537,7 +1542,7 @@ private Chunk chunkUUID() {
       if(idx >= 0)i = idx;
       else cancel_sparse(); // for now don't bother setting the sparse value
     }
-    _is[i] = _sslen;
+    if (_is != null) _is[i] = _sslen;
```

Comment:
_ss actually can be null in case all elements were missing, _is can not. Furthermore, it does not lead to a broken set operation - the set will be performed as it should.

On the other hand, adding null check for the _is ptr here would lead to a broken state where there is a string appended to the end  but no pointer to it. Effectively the update would be  silently dropped. Better to NPE.